framework_tvb/tvb/tests/framework/core/services/user_service_test.py

```diff
@@ -259,7 +259,7 @@ def test_check_login_bad_user(self):
         """
         Flow for entering a bad/invalid username.
         """
-        user = model.User("test_user", md5("test_pass").hexdigest(), "test_mail@tvb.org", True, "user")
+        user = model_project.User("test_user", md5("test_pass".encode('utf-8')).hexdigest(), "test_mail@tvb.org", True, "user")
```

Comment:
I see this code repeated multiple times in the code (tests abd core code)
`md5(pass_string.encode('utf-8')).hexdigest()`
We should have one function reused. Maybe put it in tvb.core.utils.py